notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Is there a way to run a command to switch the kernel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>It seems it could be possible by kicking some javascript functions directly from cell, but I think it would be too complicated I think.</p>